templates/testpmd/resource-specs/sriov-vm1-template.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: dpdk-vm-template
+objects:
+- apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    annotations:
+      openshift.io/display-name: "RHEL VM with SR-IOV-backed NIC and DPDK."
+      description: >-
+        Template for RHEL 8.4 VM, with a secondary interface backed by an SR-IOV node NIC.
+        Involves replacing the default NIC driver (iavf) to vfio-pci, to enable running DPDK.
+      openshift.io/documentation-url: "https://github.com/RHsyseng/cnv-supplemental-templates/tree/main/templates/testpmd"
+    name: ${NAME}
+    namespace: ${NAMESPACE}
+    labels:
+      app: ${NAME}
+      vm.kubevirt.io/template: dpdk-vm-template
+  spec:
+    dataVolumeTemplates:
+    - apiVersion: cdi.kubevirt.io/v1beta1
+      kind: DataVolume
+      metadata:
+        name: ${NAME}
```

Comment:
nit: I would call it DV_NAME

Reply:
I would as well, and I actually did call it DV_NAME at first, but turns out that all other Openshift Virtualization templates with DV use `NAME` for the DV name, and we want this template to be aligned with them.

Reply:
I noticed that we keep parameter name explicit so I thought DV name should be expressed explicitly as well,
In general I think that the other examples in a code base are also need to put to the test, the fact there are few
similar examples doesn't mean its correct :slightly_smiling_face:.
Having said that, I understand its just for the template "internal use" so I guess its fine..